ui/app/routes/vault/cluster/secrets/backend/secret-edit.js

```diff
@@ -102,7 +102,7 @@ export default Route.extend(UnloadModelRoute, {
       pki: secret && secret.startsWith('cert/') ? 'pki-certificate' : 'role-pki',
       cubbyhole: 'secret',
       kv: backendModel.get('modelTypeForKV'),
-      keymgmt: `keymgmt/${transition.to.queryParams.itemType || 'key'}`,
+      keymgmt: `keymgmt/${options.queryParams.itemType || 'key'}`,
```

Comment:
Should we throw in a ternary here so it doesn't fail when the keys list has no query params?

Reply:
Sure I can but I'm pretty sure `queryParams` is always an object when coming from the `transition`.

Reply:
Ok I added some additional checks to ensure there isn't a type error when accessing `itemType` on `queryParams`.